client/web/compose/src/components/Admin/Module/UniqueValues.vue

```diff
@@ -0,0 +1,210 @@
+<template>
+  <div>
+    <div class="d-flex justify-content-end">
```

Comment:
I know i said to do this but had a different idea how it would look.
Lets just move it below all the values with a "<hr>" above it, to separate it from the constraints
Also its + Add new constraint